doc/nrf/gs_ins_windows.rst

```diff
@@ -391,8 +391,13 @@ Setting up the command line build environment
 If you want to build and program your application from the command line, you must set up your build environment by defining the required environment variables every time you open a new |bash|.
 
 To define the environment variables, navigate to the ``ncs`` folder and enter the following command: |envfile|
+For more information on the various relevant environment variables, see :ref:`zephyr:env_vars_important`.
 
 If you need to define additional environment variables, create the file |rcfile| and add the variables there.
 This file is loaded automatically when you run the above command.
 
+Also make sure that nrfjprog (part of `nRF Command Line Tools`_) is installed and the path is added to the environment variables.
+West command programs the board by using nrfjprog by default.
```

Comment:
```suggestion
The west command programs the board by using nrfjprog by default.
```

We shouldn't capitalize 'west', its name is lower case.

Reply:
Changed.